charts/opensearch/values.yaml

```diff
@@ -104,6 +104,10 @@ config:
 extraEnvs: []
 #  - name: MY_ENVIRONMENT_VAR
 #    value: the_value_goes_here
+# 2.12 onwards a custom strong password needs to be provided in order to setup demo admin user.
+# Cluster will not spin-up without this unless demo config install is disabled.
```

Comment:
Shall we highlight this in the README of this repo? in [Notes About Default Installation](https://github.com/opensearch-project/helm-charts?tab=readme-ov-file#notes-about-default-installation)

Reply:
Good call-out. I will push an update